jenkins/internal-ci/generic/pr/component_test/Motr_Pre_merge_DTM_Integration.groovy

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env groovy
+// CLEANUP REQUIRED
+pipeline { 
+    agent {
+        node {
+            label "docker-${OS_VERSION}-node"
+        }
+    }
+
+    options { 
+        skipDefaultCheckout()
+        timeout(time: 180, unit: 'MINUTES')
+        timestamps()
+        disableConcurrentBuilds()
+        ansiColor('xterm')  
+    }
+
+    parameters {  
+	    string(name: 'MOTR_REPO', defaultValue: 'https://github.com/Seagate/cortx-motr', description: 'Repo for Motr')
+        string(name: 'MOTR_BRANCH', defaultValue: 'main', description: 'Branch for Motr')
+        }
+
+    environment {
+
+        // Motr Repo Info
+
+        GPR_REPO = "https://github.com/${ghprbGhRepository}"
+        MOTR_URL = "${ghprbGhRepository != null ? GPR_REPO : MOTR_URL}"
+        MOTR_BRANCH = "${sha1 != null ? sha1 : MOTR_BRANCH}"
+
+        MOTR_GPR_REFSEPEC = "+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*"
+        MOTR_BRANCH_REFSEPEC = "+refs/heads/*:refs/remotes/origin/*"
+        MOTR_PR_REFSEPEC = "${ghprbPullId != null ? MOTR_GPR_REFSEPEC : MOTR_BRANCH_REFSEPEC}"
+
+        //////////////////////////////// BUILD VARS //////////////////////////////////////////////////
+        // OS_VERSION and COMPONENTS_BRANCH are manually created parameters in jenkins job.
+        
+        COMPONENT_NAME = "motr".trim()
+        BRANCH = "${ghprbTargetBranch != null ? ghprbTargetBranch : COMPONENTS_BRANCH}"
```

Comment:
Remove `BRANCH` variable as it is not getting used

Reply:
Done